client/my-sites/site-settings/form-security.jsx

```diff
@@ -0,0 +1,80 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { flowRight, partialRight, pick } from 'lodash';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import wrapSettingsForm from './wrap-settings-form';
+import SectionHeader from 'components/section-header';
+import Button from 'components/button';
+import Sso from './sso';
+import QueryJetpackModules from 'components/data/query-jetpack-modules';
+
+class SiteSettingsFormSecurity extends Component {
+	renderSectionHeader( title, showButton = true ) {
+		const { isRequestingSettings, isSavingSettings, translate } = this.props;
+		return (
+			<SectionHeader label={ title }>
+				{ showButton &&
+					<Button
+						compact
+						primary
+						onClick={ this.props.handleSubmitForm }
+						disabled={ isRequestingSettings || isSavingSettings }>
+						{ isSavingSettings ? translate( 'Saving…' ) : translate( 'Save Settings' ) }
+					</Button>
+				}
+			</SectionHeader>
+		);
+	}
+
+	render() {
+		const {
+			fields,
+			handleToggle,
+			jetpackSettingsUISupported,
+			handleSubmitForm,
+			isRequestingSettings,
+			isSavingSettings,
+			siteId,
+			translate
+		} = this.props;
+
+		if ( ! jetpackSettingsUISupported ) {
+			return null;
+		}
+
+		return (
+			<form
+				id="site-settings"
+				onSubmit={ handleSubmitForm }
+				className="site-settings__security-settings"
+			>
+				<QueryJetpackModules siteId={ siteId } />
+
+				{ this.renderSectionHeader( translate( 'WordPress.com log in' ) ) }
```

Comment:
I have no preferences on this one, really. I've just followed what's on p6TEKc-HQ-p2 - @rickybanister @MichaelArestad what do you think we should be using here?